### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by mmorhun**

### CODE REVIEW

**Review by rcerven — APPROVED** [review]

File: main.go [hunk 1]
```diff
@@ -90,12 +91,20 @@ func main() {
...
+	}
+	metricsOpts := server.Options{
+		BindAddress: metricsAddr,
+	}
+
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+		Client:                 clientOpts,
 		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
```
Comment (chmeliik) [inline main.go:96]: We don't need to set the port anymore?

Comment (mmorhun) [reply main.go:96]: It's scaffolded deprecated configuration for webhook port which is not used in Image Controller.

**Review by chmeliik — APPROVED** [review]
LGTM :+1:

**Pull request merged**